lib/src/differs.dart

```diff
@@ -0,0 +1,31 @@
+library observable.src.differs;
```

Comment:
Not yet if you are using `part` 👎 

Added the header though, thanks. Wished the presubmit checked that!